cmd/plan.go

```diff
@@ -118,7 +119,7 @@ func planInstall(rv cran.RVersion) (*cran.PkgDb, gpsr.InstallPlan) {
 			if configlib.IsCustomizationSet("Type", pkgSettings, pkg) {
 				err := cdb.SetPackageType(pkg, v.Type)
 				if err != nil {
-					log.WithFields(log.Fields{
+					log.WithFields(logrus.Fields{
```

Comment:
hmm there is some inconsistencies with your naming, above I saw you refactoring logrus.Fields to log.Fields (fine), but there going in the opposite direction. I'd pick one and stick to it

Reply:
So that was intentional -- I thought that, since the logger package is the only one really dealing directly with logrus, I would keep the full name there to avoid confusion. I can still change it if you want, but I prefer it this way.

Reply:
Ooh, nevermind, just took another look. I'll loook into this